alphastats/DataSet_Preprocess.py

```diff
@@ -155,26 +158,43 @@ def _imputation(self, method: str):
         )
         self.preprocessing_info.update({"Imputation": method})
 
+    def _linear_normalization(self, array):
+        """Normalize data using l2 norm without breaking when encoutering nones
+        l2 = sqrt(sum(x**2))
+
+        Args:
+            array (pd.Series): array to normalize (1D array)
+
+        Returns:
+            np.array: normalized array
+        """
+        square_sum_per_row = array.pow(2).sum(axis=1, skipna=True)
+
+        l2_norms = np.sqrt(square_sum_per_row)
```

Comment:
you could check how https://numpy.org/doc/stable/reference/generated/numpy.linalg.norm.html works with `NaN`s

Reply:
Couldn't get it to work properly, I don't think it works well with `NaN`s